package-lock.json

```diff
@@ -4373,7 +4373,7 @@
         },
         "string_decoder": {
           "version": "1.1.1",
-          "resolved": "https://registry.npmjs.org/string_decoder/-/string_decoder-1.1.1.tgz",
+          "resolved": "http://registry.npmjs.org/string_decoder/-/string_decoder-1.1.1.tgz",
```

Comment:
change to http will incur the risk of man-in-middle attack

Reply:
Thank you! This problem happened by npm registry API. I will change `http://` to `https://`.

<img width="766" alt="2018-12-15 23 00 39" src="https://user-images.githubusercontent.com/15062473/50043753-43556680-00bd-11e9-9f32-3ba6d0c02e43.png">